modules/_labs/core/react/lib/typeTest.ts

```diff
@@ -0,0 +1,139 @@
+import styled from '@emotion/styled';
+import {typeColors} from '@workday/canvas-colors-web';
+import {CSSProperties, fontFamily} from '@workday/canvas-kit-react-core';
+import {default as beta_Type} from './type';
+
+export interface CanvasTypeHierarchy {
+  level10: CSSProperties;
+  level9: CSSProperties;
+  level8: CSSProperties;
+  level7: CSSProperties;
+  level6: CSSProperties;
+  level5: CSSProperties;
+  level4: CSSProperties;
+  level3: CSSProperties;
+  level2: CSSProperties;
+  level1: CSSProperties;
+  [key: string]: CSSProperties;
+}
+
+const broswerDefaultFontSize = 16;
+/**
+ * Converts px values into base 16 rems
+ */
+const rem = (pxValue: number) => {
```

Comment:
This feels very similar to Polish's [rem](https://polished.js.org/docs/#rem) function.